examples/tutorials/trainer_api/train.py

```diff
@@ -0,0 +1,137 @@
+from determined import pytorch
```

Comment:
question:  Why not just change our existing PyTorchTrial examples as part of this change?

(not specifically part of this PR, but I mean: why have 1 example of the Right Way and 25 examples of the Old Way in our examples, rather than just 25 examples of the Right Way?)

Reply:
well, i think there's still an open question as to whether pytorch trainer is initially released as an internal tool or a first-class feature. but i definitely agree that once it becomes a public feature, we should change all the examples.

Reply:
Ok, but in either case, we should not land this example, right?

Reply:
hm. i guess we don't have a home for "internal examples". but i think this example will be helpful for dogfooding, especially local training mode. maybe it belongs in some sort of "experimental" examples directory?

Reply:
We are already drowning in examples... let's not land any that aren't strictly necessary.